tensorboard/plugins/graph/tf_graph_common/node.ts

```diff
@@ -650,32 +676,21 @@ export function getFillForNode(templateIndex, colorBy,
         // Return the hue for unknown device.
         return colorParams.UNKNOWN;
       }
-      let id = renderInfo.node.name;
```

Comment:
It'd be nice to also deduplicate the code for the TPU compatibility logic below - it looks like it was essentially copy-pasted between the device color and TPU compat color cases previously, so now that you've factored it out it would ideally be factored out in all three places.

I think it should be pretty easy to adapt, but if you'd rather not bother that's fine and one of us can clean it up later.

Reply:
Done.